src/components/Calculator.js

```diff
@@ -1,31 +1,42 @@
-        <button type="button" className="gray-btn">AC</button>
-        <button type="button" className="gray-btn">+/-</button>
-        <button type="button" className="gray-btn">%</button>
-        <button type="button" className="orange-btn">/</button>
-        <button type="button" className="gray-btn">7</button>
-        <button type="button" className="gray-btn">8</button>
-        <button type="button" className="gray-btn">9</button>
-        <button type="button" className="orange-btn">*</button>
-        <button type="button" className="gray-btn">4</button>
-        <button type="button" className="gray-btn">5</button>
-        <button type="button" className="gray-btn">6</button>
-        <button type="button" className="orange-btn">-</button>
-        <button type="button" className="gray-btn">1</button>
-        <button type="button" className="gray-btn">2</button>
-        <button type="button" className="gray-btn">3</button>
-        <button type="button" className="orange-btn">+</button>
-        <button type="button" className="gray-btn">0</button>
-        <button type="button" className="gray-btn">.</button>
-        <button type="button" className="orange-btn">=</button>
+        <div className="calc-display">{next || total || 0}</div>
+        <button type="button" onClick={this.handleClick} className="gray-btn">AC</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">+/-</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">%</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">÷</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">7</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">8</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">9</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">*</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">4</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">5</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">6</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">-</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">1</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">2</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">3</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">+</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">0</button>
+        <button type="button" onClick={this.handleClick} className="gray-btn">.</button>
+        <button type="button" onClick={this.handleClick} className="orange-btn">=</button>
```

Comment:
- [ ] While I can perform a few operations like addition, division and subtraction, but I cannot run multiplcation and division by zero. When I run it the program cracks. Please work on it and request another review.